test/resources/validate/dockerfile-from-ce-image/context/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM mcr.microsoft.com/azureml/curated/acpt-pytorch-1.11-cuda11.3:{{latest-image-tag}}
```

Comment:
Consider also adding another test case where there's a comment line above the `FROM` one.